armi/reactor/converters/uniformMesh.py

```diff
@@ -1317,7 +1317,7 @@ def paramSetter(block, vals, paramNames):
             if val is None:
                 continue
 
-            if isinstance(val, (list, numpy.ndarray)):
+            if isinstance(val, (tuple, range, list, numpy.ndarray)):
```

Comment:
Is there an expectation that val would ever be a `range` object?

Reply:
It came up in a unit test, but I fixed the unit test so I should just remove it